libs/webb-ui-components/src/constants/index.ts

```diff
@@ -61,8 +61,7 @@ export const TANGLE_DOCS_URL = 'https://docs.tangle.tools';
 export const TANGLE_GITHUB_URL = 'https://github.com/webb-tools/tangle';
 export const WEBB_DOCS_URL = 'https://docs.webb.tools';
 export const WEBB_BLOG_URL = 'https://blog.webb.tools';
-export const WEBB_TANGLE_DOCS_URL =
-  'https://docs.webb.tools/docs/tangle-network/overview/';
+export const WEBB_TANGLE_DOCS_URL = 'https://docs.tangle.tools/docs/';
```

Comment:
These links are worth looking into, if I recall correctly, some of them are outdated/down. I will be checking them soon and making an issue+PR for it if necessary.